ynput/OpenPype
# Maya: Light family - OP-1430

## Changelog Description
Introduces new `light` family in Maya for publishing lights.

## Testing notes:
1. Add any light types into `project_settings/maya/publish/ValidateLightRequired/light_types` that needs to be considered as a `light` family.
2. Add any required attribute values into `project_settings/maya/publish/ValidateLightRequired/attribute_values`. First field is `{node_type}.{attribute_name}`, second field is the value expected.
3. Add any optional attribute values into `project_settings/maya/publish/ValidateLightOptional/attribute_values`. First field is `{node_type}.{attribute_name}`, second field is the value expected.
4. Publish and load.
